.github/workflows/test_build_deploy.yml

```diff
@@ -323,6 +341,66 @@ jobs:
           name: webapp-dist
           path: ./server/dist
 
+  update_helm_chart:
+    name: Update Helm chart
+    runs-on: ubuntu-latest
+
+    needs: test_build_deploy
+
+    steps:
+      - name: Obtaining release information artifact
+        id: release-info-artifact
+        uses: actions/download-artifact@v2
+        continue-on-error: true
+        with:
+          name: release-info.json
+      - name: Indicating whether release info exist
+        id: release-info-file
+        env:
+          ARTIFACT_LOCAL_PATH: '${{ steps.release-info-artifact.outputs.download-path }}/release-info.json'
+        shell: bash
+        run: |
+          test -s "${ARTIFACT_LOCAL_PATH}" && echo '::set-output name=exists::true'
+          echo "::set-output name=releaseInfo::$(cat ${ARTIFACT_LOCAL_PATH})"
+
+      - name: Checking out 'wire-server'
+        uses: actions/checkout@v2
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        with:
+          repository: 'wireapp/wire-server'
+          fetch-depth: 1
+
+      - name: Changing Helm value of the webapp chart
+        id: change-helm-value
+        if: ${{ steps.release-info-file.outputs.exists == 'true' }}
+        shell: bash
+        run: |
+          sed --in-place --expression="s/  tag: .*/  tag: \"${{ fromJSON(steps.release-info-file.outputs.releaseInfo).imageTag }}\"/" ./charts/webapp/values.yaml
```

Comment:
```suggestion
          sed -i -e "s/  tag: .*/  tag: \"${{ fromJSON(steps.release-info-file.outputs.releaseInfo).imageTag }}\"/" ./charts/webapp/values.yaml
```

Reply:
using the short versions of the flag would decrease readability. I'll stick with the long ones :)

Reply:
Okay, but why are you using the long flags everywhere else then? ;)